src/go/rpk/pkg/cli/cluster/partitions/move_status.go

```diff
@@ -0,0 +1,216 @@
+					PartitionSize  string
+					BytesMoved     string
+					BytesRemaining string
+				}{
+					nt,
+					rr.PartitionID,
+					oldReplica,
+					newReplica,
+					completion,
+					sizeFn(rr.PartitionSize),
+					sizeFn(rr.BytesMoved),
+					sizeFn(rr.BytesLeft),
+				}
+			}
+
+			types.Sort(response)
+
+			const (
+				secMove      = "Partition movements"
+				secReconcile = "Reconciliation statuses"
+			)
+			sections := out.NewSections(
+				out.ConditionalSectionHeaders(map[string]bool{
+					secMove:      true, // we always print this section
+					secReconcile: all,  // we only print this section if -a is passed
+				})...,
+			)
+			sections.Add(secMove, func() {
+				headers := []string{"Namespace-Topic", "Partition", "Moving-from", "Moving-to", "Completion-%", "Partition-size", "Bytes-moved", "Bytes-remaining"}
+				tw := out.NewTable(headers...)
+				defer tw.Flush()
+				for _, tps := range response {
+					tw.PrintStructFields(f(&tps))
+				}
+			})
+
+			sections.Add(secReconcile, func() {
+				var j int
+				for _, p := range response {
+					fmt.Printf("%s\n", p.Ns+"/"+p.Topic+"/"+strconv.Itoa(p.PartitionID))
+					headers := []string{"Node-id", "Core", "Type", "Retry-number", "Revision", "Status"}
+					tw := out.NewTable(headers...)
+					for _, rs := range p.ReconciliationStatuses {
+						var row []interface{}
+						row = append(row, rs.NodeID)
+						for _, s := range rs.Operations {
+							row = append(row, s.Core, s.Type, s.RetryNumber, s.Revision, s.Status)
+						}
+						tw.Print(row...)
+					}
+					tw.Flush()
+					j++
+					if j < len(response) {
+						fmt.Println()
+					}
+				}
+			})
+		},
+	}
+
+	cmd.Flags().BoolVarP(&all, "print-all", "a", false, "Print internal states about movements for debugging")
+	cmd.Flags().BoolVarP(&human, "human-readable", "H", false, "Print the partition size in a human-readable form")
+	cmd.Flags().StringSliceVarP(&partitions, "partition", "p", nil, "Partitions to print the ongoing movements (repeatable)")
+
+	return cmd
+}
+
+// This function returns true when a partition that movement is
+// ongoing is a requested partition by the --partition option.
+func contains(pReq []string, pRes string) bool {
+	for _, p := range pReq {
+		if p == pRes {
+			return true
+		}
+	}
+	return false
+}
+
+const helpListMovement = `Show ongoing partition movements.
+
+By default this command lists all the ongoing partition movements in the cluster.
+Topics can be specified to print the move status of specific topics. By default,
+this command assumes the "kafka" namespace, but you can use a "namespace/" to
+specify internal namespaces.
+
+    rpk cluster partitions move-status
+    rpk cluster partitions move-status foo bar kafka_internal/tx
+
+The "--partition / -p" flag can be used with topics to additional filter
+requested partitions:
+
+    rpk cluster partitions move-status foo bar --partition 0,1,2
+
+The output contains the following columns, where PARTITION-SIZE is in bytes.
```

Comment:
```suggestion
The output contains the following columns with PARTITION-SIZE in bytes.
```

Reply:
Fixed in a new commit: 8c76b5f62b12ebc5a1affae0971a435c475d18ac